src/main/groovy/com/readytalk/gradle/plugins/ci/CiPublishingPlugin.groovy

```diff
@@ -20,8 +20,9 @@ class CiPublishingPlugin implements Plugin<Project>, PluginUtils {
 
   void apply(final Project project) {
     this.project = project
+    project.plugins.apply(BasePlugin)
 
-    withAnyPlugin(['ivy-publish', 'maven-publish']) {
+    withAnyPlugin(['ivy-publish', 'maven-publish', 'com.gradle.plugin-publish']) {
```

Comment:
Word of caution - `withAnyPlugin` applies the closure for each plugin in the list. I don't think it's an issue in this case, but something to be aware of. It's very unlikely that a project will use both maven and ivy publishing, but maven and plugin-publish is very likely to be combined.

It turns out this is very difficult to work around due to having to be very careful how I pass the closures to Gradle.